pertpy/tools/_perturbation_space/_simple.py

```diff
@@ -59,9 +62,106 @@ def __call__(self, adata: AnnData, embedding_key: str = "X_umap", *args, **kwarg
         return ps_adata
 
 
-class PseudobulkSpace:
+class PseudobulkSpace(PerturbationSpace):
```

Comment:
Why is this whole thing a custom implementation and not using decoupler? https://decoupler-py.readthedocs.io/en/latest/notebooks/pseudobulk.html

Reply:
solved now, I didn't even know what decoupler was